base/file.jl

```diff
@@ -356,13 +339,33 @@ is an open file object for this path.
 mktemp(parent)
 
 """
-    mktempdir(parent=tempdir())
+    mktempdir(parent=tempdir(); prefix="$temp_prefix")
 
 Create a temporary directory in the `parent` directory and return its path.
-If `parent` does not exist, throw an error.
+If `parent` does not exist, throw an error. An optional `prefix` to the directory name can
```

Comment:
The wrapping seems odd here. If you wrap once at the period, I think the rest will fit on the following line.

Reply:
Fixed in latest check-in.